index.bs

```diff
@@ -7422,23 +8777,39 @@ partial dictionary MLOpSupportLimits {
 ### slice ### {#api-mlgraphbuilder-slice}
 Produce a slice of the input tensor.
 <script type=idl>
+dictionary MLSliceOptions : MLOperatorOptions {
+  sequence<[EnforceRange] unsigned long> strides;
+};
+
 partial interface MLGraphBuilder {
   MLOperand slice(MLOperand input,
                   sequence<[EnforceRange] unsigned long> starts,
                   sequence<[EnforceRange] unsigned long> sizes,
-                  optional MLOperatorOptions options = {});
+                  optional MLSliceOptions options = {});
 };
 
-partial  dictionary MLOpSupportLimits {
+partial dictionary MLOpSupportLimits {
   MLSingleInputSupportLimits slice;
 };
 </script>
+
+{{MLSliceOptions}} has the following members:
+<dl dfn-type=dict-member dfn-for=MLSliceOptions>
+    : <dfn>strides</dfn>
+    ::
+        The stride to step over each input along each axis.
+        The length of the strides array must equal the [=MLOperand/rank=] of the input tensor.
+        The the default is an array of length [=MLOperand/rank=] consisting of all 1's.
+        e.g. [1,1,1] for a 3-D tensor.
+        Strides must be greater than zero.
+</dl>
+
 <div dfn-for="MLGraphBuilder/slice(input, starts, sizes, options)" dfn-type=argument>
     **Arguments:**
         - <dfn>input</dfn>: an {{MLOperand}}. The input tensor.
-        - <dfn>starts</dfn>: [=sequence=]<{{unsigned long}}>. The starting index to slice of each input dimension, of length N where N is the [=MLOperand/rank=] of the input tensor. For each dimension *d* of {{MLGraphBuilder/slice(input, starts, sizes, options)/input}}, {{MLGraphBuilder/slice(input, starts, sizes, options)/starts}}[*d*] indicates the starting index to slice in that dimension. The starting index must be in the range [0, input size - 1] in that dimension.
-        - <dfn>sizes</dfn>: a [=sequence=]<{{unsigned long}}>. The number of elements to slice of each input dimension, of length N where N is the [=MLOperand/rank=] of the input tensor. For each dimension *d* of {{MLGraphBuilder/slice(input, starts, sizes, options)/input}}, {{MLGraphBuilder/slice(input, starts, sizes, options)/sizes}}[*d*] indicates the number of elements to slice in that dimension. The size must not be 0 and must satisfy the constraint `starting index + size <= input size` in that dimension.
-        - <dfn>options</dfn>: an {{MLOperatorOptions}}. Specifies the optional parameters of the operation.
+        - <dfn>starts</dfn>: an [=sequence=]<{{unsigned long}}>. The starting index to slice of each input dimension, of length N where N is the [=MLOperand/rank=] of the input tensor. For each dimension *d* of {{MLGraphBuilder/slice(input, starts, sizes, options)/input}}, {{MLGraphBuilder/slice(input, starts, sizes, options)/starts}}[*d*] indicates the starting index to slice in that dimension. The starting index must be in the range [0, input size - 1] in that dimension.
```

Comment:
nit: "an sequence" sounds wrong; I guess if you pronounce it "an unsigned long sequence" it works, but that's not how other examples in the spec are written today

Reply:
🤦‍♂️